README.md

```diff
@@ -4,14 +4,22 @@ dev-setup
 
 ## Utilization
 
-1. Download and install Xcode
+1. Go to the AppStore and Download and install Xcode
 2. Open Xcode and accept its terms
-3. Run script
+3. Clone this repository using the Terminal:
 ```
+ git clone https://github.com/RockSpoon/dev-setup.git
+
+```
+and then run:
+```
+cd dev-setup/ 
+ls
+sudo chmod -R $(whoami) /usr/local/Homebrew 
```

Comment:
This comand is wrong. It should be chown.
Also, you only had to do it because the computer was used by others before you.